.woodpecker/.test-pr.yml

```
@@ -0,0 +1,16 @@
+steps:
+  install:
+    image: danlynn/ember-cli:4.8.0
+    commands:
+      - npm ci
```

It seems this project doesn't use lock files yet.

```suggestion
      - npm install
```